frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -1,17 +1,43 @@
 <script setup lang="ts">
+import { computed } from 'vue';
 import LabelText from '@/components/design-system/typography/LabelText.vue';
 
 export interface Props {
   active?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const classes = computed(() => ({
+  'pill--active': props.active,
+}));
+
+const props = withDefaults(defineProps<Props>(), {
   active: false,
 });
 </script>
 
 <template>
-  <LabelText>
-    <slot />
-  </LabelText>
+  <div :class="['pill', classes]">
```

Comment:
Pokud naplánuješ přidat několik dalších tříd, tak bych řádek 10 dal sem místo `classes`, ať se třídy řeší jen na jednom místě a není to roztahaný po souboru.

Reply:
Mně tohle naopak přijde dobrý, u toho elementu mít vždy jen tu primární classu a zbytek vždycky definovat nahoře, i když tam bude třeba jenom jedna třída, ať je to ve všech komponentách stejně.

Reply:
Protože jsem viděla, že se to takhle už někde používá, tak to tedy nechávám.